targets/TARGET_STM/sleep.c

```diff
@@ -35,6 +35,100 @@
 extern void HAL_SuspendTick(void);
 extern void HAL_ResumeTick(void);
 
+/*  Wait loop - assuming tick is 1 us */
+static void wait_loop(uint32_t timeout)
+{
+    uint32_t t1, t2, elapsed = 0;
+    t1 = us_ticker_read();
+    do {
+        t2 = us_ticker_read();
+        elapsed = (t2 > t1) ? (t2 - t1) : ((uint64_t)t2 + 0xFFFFFFFF - t1 + 1);
+    } while (elapsed < timeout);
+    return;
+}
+
+// On L4 platforms we've seen unstable PLL CLK configuraiton
+// when DEEP SLEEP exits just few �s after being entered
+// So we need to force MSI usage before setting clocks again
+static void ForceClockOutofDeepSleep(void) {
```

Comment:
`{` new line as the rest of the functions

Reply:
ok will do